Fenced_Frames_Ads_Reporting.md

```diff
@@ -109,7 +109,7 @@ Note `window.fence` here is a new namespace for APIs that are only available fro
 
 ## registerAdBeacon
 
-A similar API was initially discussed here: https://github.com/WICG/turtledove/issues/99 for reporting clicks. The idea is that the buyer and seller side worklets are able to register a URL with the browser in their reportWin and reportResult APIs. A beacon will be sent to the registered URL when events are reported by the fenced frame via reportEvent.
+A similar API was initially discussed here: https://github.com/WICG/turtledove/issues/99 for reporting clicks. The idea is that the buyer and seller side worklets are able to register a URL with the browser in their reportWin and reportResult APIs. The registered URL is required to be attested for Protected Audience API. Please see [the Privacy Sandbox enrollment attestation model](https://github.com/privacysandbox/attestation#the-privacy-sandbox-enrollment-attestation-model). If the URL is attested for Protected Audience API, a beacon will be sent to the registered URL when events are reported by the fenced frame via reportEvent. Otherwise the beacon will not be sent.
```

Comment:
"registered URL's site is required..."

Reply:
I think it will be good to give an example here as well.

Reply:
Updated the wording. About the example, it already [has one down below](https://github.com/WICG/turtledove/pull/702/files#diff-6857dacb072323c2d230a4c8c1ff0fe74ae295da2e5e7af00f84c1352a066a53R121). Is this sufficient or would you like adding more?

Reply:
Added this below the example code.

Reply:
Let's also add a statement about redirects:

"If the destination decides to redirect, it's that site's responsibility to share any data in the redirect URL as per the attestation guarantees. The redirected site is not checked for enrollment and attestation since the browser doesn't add any data directly to those redirects' URL."

Reply:
Added.

Reply:
Can this thread be resolved?